### PR TITLE
Remove Cutlass and Tanager; add Rumskib

### PR DESCRIPTION
**Pull request by TheGiraffe3**
I've merged the original two plugins into one.
Rumskib is Danish for "ship".

### CODE REVIEW

**Comment by Hecter94** [comment]
Should we keep the two old files as well as the new one? I typically do like to keep old versions even if they're obsoleted.

**Comment by TheGiraffe3** [comment]
Your choice. I am deleting the old repositories, so I set the directlink to N/A. I assume that's how you'd like it?

**Comment by zuckung** [comment]
for maximal informations, you could put in status that these plugins are part the new one.

File: res/pluginlist/Cutlass.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 author=TheGiraffe3
-website=https://github.com/TheGiraffe3/es-cutlass
-directlink=https://github.com/TheGiraffe3/es-cutlass/archive/refs/tags/v1.0.0.zip
+website=N/A
+directlink=N/A
 category=Ships
 status=Complete
 description=Adds a new ship called the Cutlass, the pirate version of a Gunboat.
```
Comment (TheGiraffe3) [inline res/pluginlist/Cutlass.txt:6]:
```suggestion
description=Adds a new ship called the Cutlass, the pirate version of a Gunboat. This plugin is deprecated; has been replaced by the Rumskib plugin.
```

File: res/pluginlist/Tanager.txt [hunk 2]
```diff
@@ -1,6 +1,6 @@
 author=TheGiraffe3
-website=https://github.com/TheGiraffe3/es-tanager
-directlink=https://github.com/TheGiraffe3/es-tanager/archive/refs/heads/main.zip
+website=N/A
+directlink=N/A
 category=Ships
 status=Complete
 description=Adds a new ship whose stats are a cross between that of the Shuttle and the Scout.
```
Comment (TheGiraffe3) [inline res/pluginlist/Tanager.txt:6]:
```suggestion
description=Adds a new ship whose stats are a cross between that of the Shuttle and the Scout. This plugin is deprecated; has been replaced by the Rumskib plugin.
```

**Pull request merged**